pkg/kp/constants.go

```diff
@@ -23,3 +24,7 @@ func RealityPath(args ...string) string {
 func HookPath(hookType hooks.HookType, args ...string) string {
 	return strings.Join(append([]string{HOOK_TREE, hookType.String()}, args...), "/")
 }
+
+func LockPath(args ...string) string {
```

Comment:
Food for thought, I thought i'd find this in lock.go. Looking at the rest of the contents of this file it makes sense though.